### ISSUE TITLE
Can I use the mAP on the validation set to evaluate	the training result？

### ISSUE THREAD

**Issue by cuppp1998** [ultralytics/yolov5#2497]
Now I have two datasets with different training sets and the same validation set. So can I use the two best mAP of the validation set to compare the two models or should I use another test set to do so?
